manual/src/cmds/native.etex

```diff
@@ -206,13 +206,48 @@ the following environment variables are also consulted:
 \begin{options}
 \item["OCAMLRUNPARAM"]  Same usage as in "ocamlrun"
   (see section~\ref{s:ocamlrun-options}), except that option "l"
-  is ignored (the operating system's stack size limit
-  is used instead).
+  is ignored.  (How to control the stack size is explained in
+  section~\ref{s:native:stacksize}.)
 \item["CAMLRUNPARAM"]  If "OCAMLRUNPARAM" is not found in the
   environment, then "CAMLRUNPARAM" will be used instead.  If
   "CAMLRUNPARAM" is not found, then the default values will be used.
 \end{options}
 
+\section{s:native:stacksize}{Size of the stack}
+
+Native-code executables produced by the "ocamlopt" compiler use the
+stack managed by the operating system to implement function calls in
+non-tail positions.  The system stack grows on demand up to a certain
+size limit maintained by the operating system.  We now explain how this
+size limit is determined and can be controlled.
+
+\begin{unix}
+Linux, macOS, and other Unix-like operating systems maintain two
+limits for the stack size: a current limit and a hard limit.  These
+limits can be modified (as long as the hard limit is not exceeded)
+with the "ulimit -a" shell command or the "setrlimit" system call.
+At start-up, executables generated by "ocamlopt" try to set a
+comfortable limit for the stack size, as follows:
+\begin{itemize}
```

Comment:
It's not clear from this description that the limit will not be lowered by the start-up code. You could add a first item stating that.

Reply:
I'm not sure this property is so important.  What matters to me is that if an explicit limit was set by the user with `ulimit`, it is always honored.